scripts/bygg

```diff
@@ -428,6 +428,24 @@ sync_images()
     return 1
   fi
 
+  IFS=":" read -ra destination_parts <<< "$2"
```

Comment:
It's usually a good idea to reset IFS after modifying it.

Reply:
it is a local variable. anything you want to change?

Reply:
https://bash.cyberciti.biz/guide/$IFS

Reply:
If you are sure it won't cause any problems you can keep it as it is.

Reply:
it would alread have cause it. there is 3 other place in the script that alter the IFS. thanks for pointing that out. we should investigate if we can restore it on another clenup mr